ui/app/services/control-group.js

```diff
@@ -114,20 +114,10 @@ export default Service.extend({
     return [returnedParams, qps];
   },
 
-  urlFromTransition(transitionObj) {
-    let transition = transitionObj.to;
-    let [params, queryParams] = this.paramsFromTransition(transition, [], {});
-    let url = this.router.urlFor(transition.name, ...params, {
-      queryParams,
-    });
-    return url.replace('/ui', '');
-  },
-
   handleError(error, transition) {
-    let { accessor, token, creation_path, creation_time, ttl } = error;
-    let url = this.urlFromTransition(transition);
-    let data = { accessor, token, creation_path, creation_time, ttl };
-    data.uiParams = { url };
+    const { accessor, token, creation_path, creation_time, ttl } = error;
+    const data = { accessor, token, creation_path, creation_time, ttl };
+    data.uiParams = { url: transition?.intent?.url };
```

Comment:
Will this keep all the queryparams as part of the URL? I know that can be important for the redirects (specifically in secret engines like transform)

Reply:
I tested and the query params are part of the url but now I'm wondering if the intent url is defined on the transition if the route change was initiated from within Ember. It doesn't look like it is. I'll look into this further before merging.

The reason why I changed it was that `backend` was not present in the `transition.params` object so when the control group service was building the url from the transition object the transition would ultimately error because of the missing backend param.